tests/grid/project_to_object_01.output

```diff
@@ -103,7 +103,7 @@ DEAL::trial point:                          -1.20701 -1.04224 -1.20701
 DEAL::projected point:                      -1.20701 -1.04224 -1.20701
 DEAL::trial point distance from origin:     2.00000
 DEAL::projected point distance from origin: 2.00000
-DEAL::Error:                                8.09719e-13
+DEAL::Error:                                1.41191e-09
```

Comment:
Oh, unfortunate. Do you know what happened here? There are numerous other cases here where the error become substantially larger.

Reply:
Excellent spot. This was due to roundoff when I changed `n=e2 - (e1*e2)*e1` to `n=v2-(v2*e1)*e1` (both were normalized before use). There was no reason to change this, I flipped it back and now the test is stable again.